packages/components/src/components/radio-button/_radio-button.scss

```
@@ -71,7 +71,7 @@
     flex-shrink: 0;
     width: rem(18px);
     height: rem(18px);
-    margin-right: $carbon--spacing-03;
+    margin: 0.2rem $carbon--spacing-03 0.2rem 0;
```

Would this make sense as:

```suggestion
    margin: 2px $carbon--spacing-03 2px 0;
```

Which would account for the outline size? Or is the 3.2px value exact? Wasn't sure where it was coming from 👀

---

@joshblack that makes sense! I guess I just thought we always want to use `rem` - just in general what is the rule of thumb around that? Re: when to use `rem` v `px`?

---

@andreancardona let me know if I'm missing something, but it seemed like this margin was being used to offset the outline width which was in px. If so, and we used rem, then we might go past the size of the outline if the font-size increases even though the outline width itself didn't change.